cypress/fixtures/user.json

```diff
@@ -3,5 +3,7 @@
   "password": "12345678",
   "invalidEmail": "asdasd@gmail.com",
   "invalidPassword": "asdasd",
-  "phone": "0812221111"
+  "phone": "0812221111",
```

Comment:
What the purposes of this phone number? Because I see on **Login using OTP with incorrect phone number** are using `cy.get(LoginPage.mobilePhoneField).type('839427292');`

Reply:
We are still using those phone number for login as you can see [here](https://github.com/isaactreedots/AutomatedPOC/blob/9e7b2778053c2459ef80740d88efd9d5abde1a92/cypress/integration/ui_tests/login.spec.js#L34). But it will be modify later on to use Treedots number.